app/models/apple/publisher.rb

```diff
@@ -31,7 +31,15 @@ def podcast
     end
 
     def episodes_to_sync
-      show.episodes
+      filter_episodes(show.episodes)
+    end
+
+    def filter_episodes(eps)
+      # Reject episodes if the audio is marked as uploaded/complete
+      # or if the episode is a video
+      eps
+        .reject(&:synced_with_apple?)
+        .reject(&:video_content_type?)
```

Comment:
gotcha
This is filtered out of the Apple specific episode list
so we would still have the video in the public rss feed (in the case of ad free on apple sub)
makes sense

Reply:
Not sure it matters, as this already merged - but it might be simpler to check the `episode.medium` enum now?  

```ruby
.reject(&:medium_video?)
```

Saves the trouble of iterating over the contents.